index.js

```diff
@@ -1,56 +1,45 @@
 'use strict';
 
-var forEach = require('foreach');
+var toStr = Object.prototype.toString,
```

Comment:
please use one "var" per var - this is an intentional decision so the debugger can step through them.